imap_processing/swe/tests/test_decom.py

```diff
@@ -0,0 +1,69 @@
+import unittest
```

Comment:
I personally like `pytest` style tests better than unittest. Have we decided on style as a team at all? @bourque ?

For pytest, you could make fixtures of the information you need and then use it later. Sean added these to the sds repo. https://github.com/IMAP-Science-Operations-Center/sds-data-manager/blob/development/tests/infrastructure/conftest.py

```python
@pytest.fixture
def swe_packet_list():
        packet_file = Path('imap_processing/swe/tests/science_block_20221116_163611Z_idle.bin')
        xtce_document = Path('imap_processing/swe/swe_packet_definition.xml')
        return decom_packet(packet_file, xtce_document)


def test_total_packets_in_data_file(swe_packet_list):
        total_packets = 23
        assert len(swe_packet_list) == total_packets
```

Reply:
converted test from unittest to pytest

Reply:
That would great!